tests/integration/test_script_runner.py

```diff
@@ -104,25 +104,21 @@ class BackendDependencies(enum.Enum):
         "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/how_to_choose_which_dataconnector_to_use.py",
         "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
         "data_dir": "tests/test_sets/dataconnector_docs",
-        "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
```

Comment:
`util_script` is not needed for data_connector tests

Reply:
🙇🏻